db_upgrade_script/mosip_certify/sql/0.9.1_to_0.10.0_rollback.sql

```diff
@@ -1,3 +1,6 @@
 drop table rendering_template;
 drop table credential_template;
 drop table ca_cert_store;
+
+--- Keymanager policy drop ---
+DELETE FROM certify.key_policy_def where APP_ID in ('CERTIFY_VC_SIGN_RSA', 'CERTIFY_VC_SIGN_ED25519', 'BASE');
```

Comment:
these are removed as they were originally absent in the 0.10.0 release and have been added midway